.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -1,9 +1,11 @@
-**Fixes issue:** #
+**Please provide enough information so that others can review your pull request:**
 
-**Changes proposed in this pull request:**
+<!-- You can skip this if you're fixing a typo or adding an app to the Showcase. -->
 
--
--
--
+**Explain the **details** for making this change. What existing problem does the pull request solve?**

```

Comment:
**Explain the *details* for making this change. What existing problem does the pull request solve?**

Reply:
👍 n1